### DISCUSSION TITLE
Error While setting up a basic application as per the tutorial

### DISCUSSION THREAD

**Discussion by hunainabbasi** [deepgram/community#116] [General help] [answered]
Refer below for the error:

Class 'SoundStreamPlugin' is not abstract and does not implement abstract member public abstract fun onRequestPermissionsResult(p0: Int, p1: Array<(out) String!>, p2: IntArray): Boolean defined in io.flutter.plugin.common.PluginRegistry.RequestPermissionsResultListener

**Comment by briancbarrow** [comment]
@hunainabbasi Can you provide a little more context? Are you using one of our SDKs? It looks like you are using Flutter, which tutorial were you following?

**Reply by medicalnp** [reply]
I am facing the same error aswell, I followed the documentation under your flutter documentation for Deepgram.

**Comment by jjmaldonis** [comment, marked as answer]
Hey @hunainabbasi we updated our Flutter guide + code on Monday. Here is a link to the updated blog article: https://deepgram.com/learn/flutter-speech-to-text-tutorial
